.travis.yml

```diff
@@ -64,6 +65,7 @@ matrix:
 
     - name: openblas, vtk, full dep using brew
       os: osx
+      python: "3.7"
```

Comment:
did we bump dependency here? can you check that readme.rst file is still up to date?

Reply:
Indeed the rst is not up-to-date (Actually it was already out-of-date before these changes). A problem is that the instructions might differ with the fedora versions... Now, I must say, that README.rst does not really mention python for MacOSX.... It probably needs a small rewrite to better emphasize python support. I think on ubuntu, it works with python 3.6.3, so I do not know what you mean by bump the version.

Globally, probably, even python2 still works. It is only the cmake support for detecting python2 that is not there.... But really, do we want to support python2 ???